docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md

```diff
@@ -154,7 +154,7 @@ trainer.validate(ov_q_model, dataloader)
 trainer.test(ov_q_model, dataloader)
 trainer.predict(ov_q_model, dataloader)
 ```
-Same as ONNXRuntime, it equals to converting the model from Pytorch to OpenVINO firstly and then doing quantization on the converted OpenVINO model:
+Same as quantization using INC with ONNXRuntime, it equals to converting the model from PyTorch to OpenVINO firstly and then doing quantization on the converted OpenVINO model:
```

Comment:
How about "Same as using onnxruntime accelerator" ?

Reply:
Agree :D Have changed it